src/features/user/MFV2/MyContributions/MyContributions.module.scss

```diff
@@ -308,10 +312,14 @@
 .registrationItemCard {
   display: flex;
   flex-direction: column;
+  justify-content: center;
   background-color: #ffffff;
   border: 3px solid rgba(#007a49, 0.1);
   border-radius: 12px;
   overflow: hidden;
+  width: 100%;
+  min-width: 0; // Allow shrinking below 450px
```

Comment:
Why was this added? The registration cards seem to shrink in a similar way for width<450px without specifying `min-width:0`

Reply:
These styles apply to the project card and registration cards. When this wasn't done, the cards stopped resizing as the container got narrower, and the content at the right got hidden.